test/file-reader-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const fs = require('fs');
+const expect = require('chai').expect;
+const fileReader = require('../lib/file-reader.js');
+
+describe('Reading all three files', function(){
+  // describe('read files asynchronously?', function(){
+  //   var threeFileData = [];
+  //   before( function(done){
+  //     fs.readFile(`${__dirname}/../data/one.txt`, function(err, data){
+  //       threeFileData.push(data.toString());
+  //       fs.readFile(`${__dirname}/../data/two.txt`, function(err, data){
+  //         threeFileData.push(data.toString());
+  //         fs.readFile(`${__dirname}/../data/three.txt`, function(err, data){
+  //           threeFileData.push(data.toString());
+  //           done();
+  //         });
+  //       });
+  //     });
+  //   });
+  // });
+
```

Comment:
You'll always want to make sure you removed commented out code before pushing up to the githubs :)